### PR TITLE
feat: add link to label

### PR DESCRIPTION
**Pull request by wa0x6e**
### Summary

Depends on https://github.com/snapshot-labs/sx-monorepo/pull/910

This PR enable label to link to the proposal page, filtered by the given label

### How to test

1. Go to a space proposals page with labels
2. Click on label in the proposals title
3. It should redirect to the proposals page, with the given label filter
4. Repeat same steps, for label inside the single proposal page

### CODE REVIEW

File: apps/ui/src/components/ProposalLabels.vue [hunk 1]
```diff
@@ -27,6 +29,15 @@ const validLabels = computed(() => {
       :key="label.id"
       :label="label.name"
       :color="label.color"
+      :to="
+        withLink
+          ? {
+              name: 'space-proposals',
+              params: { space: `${space.network}:${space.id}` },
+              query: { 'labels[]': label.id }
```
Comment (ChaituVR) [inline apps/ui/src/components/ProposalLabels.vue:37]: we can use `label` instead
Untitled 7

File: apps/ui/src/components/Ui/ProposalLabel.vue [hunk 2]
```diff
@@ -46,7 +50,9 @@ function checkColorProximity(color: string): {
 </script>
 
 <template>
-  <div
+  <component
+    :is="to ? AppLink : 'div'"
+    :to="to"
```
Comment (ChaituVR) [inline apps/ui/src/components/Ui/ProposalLabel.vue:55]: Maybe it would be better to handle this in AppLink. If `to` doesn't exist, it use `div` instead?

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Review by bonustrack — APPROVED** [review]
tACK

**Review by ChaituVR — APPROVED** [review]
utAck (tested it previously)

**Comment by bonustrack** [comment]
Can we merge this one? @wa0x6e

**Comment by wa0x6e** [comment]
No, the main pr was reverted, waiting for a fix (on checkpoint I think)

**Pull request merged**